README.md

```diff
@@ -26,6 +25,20 @@ version for React or Vue by importing `dist/react-without-styles.js` or
 `dist/vue-without-styles.js` accordingly. The framework-specific version contains
 all features available in the default one plus specific for particular framework wrappers.
 
+### Develop
```

Comment:
I agree that this was not covered well, but have you noticed a similar section at the end of readme? I'm proposing to update it instead in a2aa92578e3683c92ae0c45312405e66a7bda2eb

Reply:
yes, I saw that later after this commit